app/src/main/java/com/duckduckgo/app/global/DuckDuckGoApplication.kt

```diff
@@ -170,6 +174,7 @@ open class DuckDuckGoApplication : HasAndroidInjector, Application(), LifecycleO
             it.addObserver(appDaysUsedRecorder)
             it.addObserver(defaultBrowserObserver)
             it.addObserver(appEnjoymentLifecycleObserver)
+            it.addObserver(dataClearerForegroundAppRestartPixel)
         }
```

Comment:
You could use `apply` instance of `also`.

Reply:
yes, they are equivalent, but no strong preference on styles here. So for now, it's fine to keep it as it is now and focus in related changes to our initial goal. But that was a nice catch. 👍